cmd/spire-server/cli/entry/delete.go

```diff
@@ -0,0 +1,86 @@
+package entry
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+
+	"github.com/spiffe/spire/cmd/spire-server/util"
+	"github.com/spiffe/spire/proto/api/registration"
+
+	"golang.org/x/net/context"
+)
+
+type DeleteConfig struct {
+	// Address of SPIRE server
+	Addr string
+
+	// ID of the record to delete
+	EntryID string
+}
+
+// Perform basic validation
+func (dc *DeleteConfig) Validate() error {
+	if dc.Addr == "" {
+		return errors.New("a server address is required")
+	}
+
+	if dc.EntryID == "" {
+		return errors.New("an entry ID is required")
+	}
+
+	return nil
+}
+
+type DeleteCLI struct{}
+
+func (DeleteCLI) Synopsis() string {
+	return "Deletes registration entries"
+}
+
+func (d DeleteCLI) Help() string {
+	_, err := d.newConfig([]string{"-h"})
+	return err.Error()
+}
+
+func (d DeleteCLI) Run(args []string) int {
+	config, err := d.newConfig(args)
+	if err != nil {
+		fmt.Println(err.Error())
```

Comment:
Looking at how printing the error and returning 1 is repeated multiple times, I was thinking that maybe we can have this in a separate function? Maybe that could help to make consistent changes in that logic if needed in the future.